### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by obuchheit**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1 @@
+Famers Marketplace
```
Comment (jamcrome) [inline README.md:1]: Looks Great

**Review by jamcrome — COMMENTED** [review]
blah

File: README.md (hunk 1 above)
Comment (zagui440s) [reply README.md:1]: rabble

**Review by zagui440s — COMMENTED** [review]
more rabble

**Review by jamcrome — APPROVED** [review]

**Review by jamcrome — APPROVED** [review]

**Pull request merged**